presto-main/src/main/java/com/facebook/presto/metadata/Metadata.java

```diff
@@ -157,6 +157,12 @@
      */
     void createTable(Session session, String catalogName, ConnectorTableMetadata tableMetadata, boolean ignoreExisting);
 
+    /**
+     * Creates a temporary table with optionally partitioning requirements can be specified.
```

Comment:
nit: `Creates a temporary table with optional partitioning requirements.` I don't think we need to say "can be specified".